### DISCUSSION TITLE
Initial Table Design Plans

### DISCUSSION THREAD

**Discussion by E8GQNB** [JaksyMarci/Cherry-Blossom-Diner#19] [Show and tell]
Here is what I've used for the floor plan of the restaurant:
![floorPlan](https://user-images.githubusercontent.com/114665672/202174584-bc3644f2-ee48-4921-a14f-6fe71e174bea.png)

With a quick demo of it's current functions:
![tables](https://user-images.githubusercontent.com/114665672/202174745-7702224a-e22a-440a-bcf9-ea34ec03bbd1.gif)

All of this is only local currently.

**Comment by StavniczkiBalazs** [comment]
I really like it!
Awesome frontend design!

**Comment by JaksyMarci** [comment]
Looks great!
